function-images/tests/chained-function-serving/consumer/consumer.go

```diff
@@ -37,6 +37,9 @@ import (
 	pb "tests/chained-functions-serving/proto"
 
 	tracing "github.com/ease-lab/vhive/utils/tracing/go"
+
+	sdk "github.com/ease-lab/vhive-xdt/sdk/golang"
```

Comment:
need to ensure that people outside of the vHive core team are able to use this benchmark. please check if the build succeeds, otherwise need to find another trick

Reply:
need to find another trick, build will not succeed since it needs private repo access. 
We will have to create separate repo for SDK.